Code/Editor/Plugins/PerforcePlugin/PerforceSourceControl.cpp

```diff
@@ -57,7 +56,7 @@ void CPerforceSourceControl::ShowSettings()
 
 void CPerforceSourceControl::SetSourceControlState(SourceControlState state)
 {
-    CryAutoLock<CryCriticalSection> lock(g_cPerforceValues);
+    AZStd::lock_guard<AZStd::mutex> lock(g_cPerforceValues);
```

Comment:
Recommendation: The lock_guard can be replaced with a scoped lock

```suggestion
    AZStd::scoped_lock lock(g_cPerforceValues);
```